app/components/elements/forms/field_component.rb

```diff
@@ -36,7 +37,22 @@ def help_text_id
         @help_text_id ||= form.field_id(field_name, 'help')
       end
 
+      def mark_required?
+        @mark_required || required
+      end
+
       def field_aria
+        {}.tap do |arias|
+          # Set aria-required if we want to indicate required, but the field
+          # does not actually have a required attribute
+          #
+          # This is used for collection/work forms where we do server-side
+          # validation and don't want to block form submission on empty fields
+          arias[:required] = true if @mark_required
+        end.merge(error_aria)
```

Comment:
I'm guessing here that we don't want the following:

```suggestion
        error_aria.tap do |arias|
          # Set aria-required if we want to indicate required, but the field
          # does not actually have a required attribute
          #
          # This is used for collection/work forms where we do server-side
          # validation and don't want to block form submission on empty fields
          arias[:required] = true if @mark_required
        end
```

because we want the `error_aria[:required]` to override the `true` here?

Reply:
This bit was originally written by @aaron-collier I think, but I think your suggestion is somewhat clearer – I don't think `error_aria` ever cares about `required`, so no need to start from an empty hash.

Reply:
:shipit: